migration/iD10/oracle/202207141111_IDEMPIERE-1963.sql

```diff
@@ -0,0 +1,620 @@
+-- IDEMPIERE-1963
+SELECT register_migration_script('202207141111_IDEMPIERE-1963.sql') FROM dual;
+
+SET SQLBLANKLINES ON
+SET DEFINE OFF
+
+-- Jul 14, 2022, 11:11:43 AM CEST
+INSERT INTO AD_Element (AD_Element_ID,AD_Client_ID,AD_Org_ID,IsActive,Created,CreatedBy,Updated,UpdatedBy,ColumnName,Name,PrintName,EntityType,AD_Element_UU) VALUES (203628,0,0,'Y',TO_TIMESTAMP('2022-07-14 11:11:43','YYYY-MM-DD HH24:MI:SS'),100,TO_TIMESTAMP('2022-07-14 11:11:43','YYYY-MM-DD HH24:MI:SS'),100,'DocSource','Document Source','Document Source','D','824512df-68c6-483a-9a63-a04e217ed86d')
```

Comment:
This is not an error, because oracle supports both functions TO_DATE and TO_TIMESTAMP, but it can be an indication that you somehow mixed up the oracle and postgresql scripts.

Normally the scripts that are generated from iDempiere have TO_DATE in oracle, and TO_TIMESTAMP in postgresql.

This script has a mix of both, it uses TO_TIMESTAMP until line 300, then it uses TO_DATE until line 534, and back to TO_TIMESTAMP.

Again, that's not an error, oracle supports it, but I'm wondering why this is happening in your commits.

Reply:
I added the lines with TO_DATE in the oracle script from an older migration script. The lines I generated lately seems to use TO_TIMESTAMP in oracle too.